nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -136,9 +136,6 @@ let
           fi
       fi
 
-    ''
-    + ''
-      exec "${startplasma-x11}"
```

Comment:
`startplasma-x11` defined above should also be removed.
https://github.com/NixOS/nixpkgs/blob/c3e415f68d20424610469e402d41a8cc0b399565/nixos/modules/services/x11/desktop-managers/plasma5.nix#L16